pkgs/top-level/python-packages.nix

```
@@ -10211,6 +10211,8 @@ self: super: with self; {
 
   schiene = callPackage ../development/python-modules/schiene { };
 
+  scholarly = callPackage ../development/python-modules/schorlarly { };
```

```suggestion
  scholarly = callPackage ../development/python-modules/scholarly { };
```

Typo causing eval fail